### ISSUE TITLE
useInteractionModality detects JAWS and NVDA as mouse in Chromium

### ISSUE THREAD

**Issue by jesusgalhub** [adobe/react-spectrum#10310]
### Provide a general summary of the issue here

Scenario: `useInteractionModality` reports "mouse" when screen reader user interacts
   GIVEN a user has started their screenreader (JAWS or NVDA) and a chromium browser (Chrome)
   WHEN the user interacts with the page clicking a button
   THEN the hook `useInteractionModality` reports the modality as "mouse"

### 🤔 Expected Behavior?

`JAWS/NVDA + Chromium` interactions are detected as `virtual`

### 😯 Current Behavior

`JAWS/NVDA + Chromium` interactions are detected as `mouse`

### 💁 Possible Solution

We created a very simple 'codepen' to log the event information for every browser and screen reader, to check if there is any difference in terms of the received information.
https://es-d-95332884020260709-019f3c39-f927-7d3c-823e-4328eff26b40.codepen.dev/

We detected that the combination `JAWS/NVDA + Chrome` throws a `click`event and a `pointerdown` event after interacting with a button by pressing `Enter`. In addition, we receive `pointerType` as `mouse`.

`JAWS/NVDA + Firefox` and `VoiceOver` does not trigger a `pointerdown` event and the `pointerType` comes as an empty string, so we can detect the device is assistive technology.

We need another type of check for failing combinations to evaluate them as `virtual` interactions.

---
## Test scenario
1- Navigate to https://codepen.io/editor/jesusgonzaleza/pen/019f3c39-f927-7d3c-823e-4328eff26b40
2- Start your screen reader
3- Interact with the "Simulate click" button
4- Check the reported data from the event

### JAWS + Chrome
Insights: a `pointerdown` event is triggered and `pointerType` is `mouse`
<img width="915" height="581" alt="Image" src="https://github.com/user-attachments/assets/71a801a6-fee4-43df-ad25-457d6a32aa32" />

### JAWS + Firefox
Insights: it works correctly. a `pointerdown` event is not triggered and `pointerType` is an empty string

<img width="851" height="577" alt="Image" src="https://github.com/user-attachments/assets/fac03bc6-dae0-4993-a660-06b8dc2bf2b2" />

### VoiceOver + Firefox
Insights: it works correctly. a `pointerdown` event is not triggered and `pointerType` is an empty string

<img width="1095" height="600" alt="Image" src="https://github.com/user-attachments/assets/2e45c187-a737-4002-8562-8db77a3ccae5" />

### VoiceOver + Chrome
Insights: it works correctly. a `pointerdown` event is not triggered and `pointerType` is an empty string

<img width="1095" height="600" alt="Image" src="https://github.com/user-attachments/assets/2e45c187-a737-4002-8562-8db77a3ccae5" />

### 🔦 Context

We detect if a user is a screen reader user in a SPA. In that case, we focus the title of the page in certain navigations so they are aware of changes.
We use `useInteractionModality` from `@react-aria/interactions` for some other component related complex accessibility logics.

### 🖥️ Steps to Reproduce

1- Start JAWS
2- Open Chrome
3- Interact with any button using the screen reader
4- Modality is reported as `mouse`

### Version

3.28.1

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows

**Comment by LFDanLu** [comment]
Verified your findings in NVDA and JAWS locally. [React](https://github.com/react/react/commit/4ef269606) (who we borrowed this logic from in the first place) used to check the coordinate of the click but seems like this virtual click detection doesn't exist at all now. We'll have to do some testing to see if there is any additional information that would distinguish JAW/NVDA + Chrome presses from standard ones

**Comment by jesusgalhub** [comment]
@LFDanLu I've commented in the PR. I believe the solution may not be sufficient for this use case

**Comment by LFDanLu** [comment]
Yeah, this will be a tricky one to solve, there very well may not be enough information from the click/pointer events to sufficiently distinguish these screen reader clicks from mouse clicks.

**Comment by jesusgalhub** [comment]
@LFDanLu 
Is this something we can report to chromium team?

**Comment by LFDanLu** [comment]
yep, definitely can, probably in the same vein as https://issuetracker.google.com/issues/40805554 which we reported for Talkback. I need to find some time to update that ticket (hopefully soon), and can see about reporting something similar for this. That being said, it may take quite some time for it to get fixed, so it would be good to figure out some kind of workaround for the short term.

**Comment by jesusgalhub** [comment]
I created this issue. Feel free to comment and add any information
https://issuetracker.google.com/issues/534943063
